corgi/core/migrations/0040_auto_20221020_1057.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.15 on 2022-10-20 10:57
```

Comment:
This migration will conflict with the existing 0040 that's already on main. Please rebase and refresh your migrations.

Reply:
oh did something slip in already ? will update

Reply:
I do not see 0040 on main ... I did conflate the migrations into one